### PR TITLE
Add api MultiplicativeDecay

### PR DESCRIPTION
**Pull request by guguguzi**
### PR types

New features
### PR changes

APIs
### Describe

Add MultiplicativeDecay, refer to [torch.optim.lr_scheduler.MultiplicativeLR](https://pytorch.org/docs/stable/generated/torch.optim.lr_scheduler.MultiplicativeLR.html?highlight=multiplicativelr#torch.optim.lr_scheduler.MultiplicativeLR)

### CODE REVIEW

**Review by jerrywgz — DISMISSED** [review]

**Review by zhwesky2010 — COMMENTED** [review]
python/paddle/fluid/dygraph/learning_rate_scheduler.py 目录下的可以不用实现，是已废弃的API目录，实现python/paddle/optimizer/lr.py中的即可

**Comment by guguguzi** [comment]
> python/paddle/fluid/dygraph/learning_rate_scheduler.py 目录下的可以不用实现，是已废弃的API目录，实现python/paddle/optimizer/lr.py中的即可

Done, thanks.

**Review by zhwesky2010 — DISMISSED** [review]

File: python/paddle/optimizer/lr.py [hunk 1]
```diff
@@ -1513,3 +1514,95 @@ def get_lr(self):
...
+                    x = paddle.uniform([10, 10])
+                    out = linear(x)
+                    loss = paddle.mean(out)
+                    loss.backward()
+                    sgd.step()
+                    sgd.clear_gradients()
+                    scheduler.step()    # If you update learning rate each step
+              # scheduler.step()        # If you update learning rate each epoch
+
+            # train on static graph mode
```
Comment (Ligoml) [inline python/paddle/optimizer/lr.py]: 静态图模式不推荐使用，这里的 example 只放动态图即可~

**Review by Ligoml — DISMISSED** [review]

File: python/paddle/optimizer/lr.py (hunk 1 above)
Comment (guguguzi) [reply python/paddle/optimizer/lr.py]: Done, thanks~

**Review by Ligoml — DISMISSED** [review]
LGTM for docs

**Review by Ligoml — DISMISSED** [review]

File: python/paddle/optimizer/lr.py [hunk 2]
```diff
@@ -1513,3 +1515,68 @@ def get_lr(self):
...
+
+        self.lr_lambda = lr_lambda
+        super(MultiplicativeDecay, self).__init__(learning_rate, last_epoch,
+                                                  verbose)
+
+    def get_lr(self):
+        if self.last_epoch > 0:
+            return self.last_lr * self.lr_lambda(self.last_epoch)
+        else:
+            return self.last_lr
```
Comment (jeff41404) [inline python/paddle/optimizer/lr.py]: accroding to description of last_epoch in doc, return self.base_lr will be better?

Comment (guguguzi) [reply python/paddle/optimizer/lr.py]: Done thanks~

**Review by jeff41404 — APPROVED** [review]
LGTM

**Review by Ligoml — APPROVED** [review]

**Review by XiaoguangHu01 — APPROVED** [review]
LG API

**Pull request merged**